doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -694,6 +695,8 @@ Libraries for Zigbee
 
       * Crypto library used for performing software AES ecryption.
         Now, the :ref:`nrfxlib:nrf_oberon_readme` is used instead of the Tinycrypt library.
+      * Optimize calling ZBOSS API in NCS platform.
+        If the ZBOSS API is called in the ZBOSS thread context, processing by workqueue is now skipped.
```

Comment:
Sorry, I missed this the first time! One tiny edit:
```suggestion
        If the ZBOSS API is called in the ZBOSS thread context, processing by the workqueue is now skipped.
```

Reply:
Done!